src/Microsoft.Health.Fhir.Anonymizer.R4.FunctionalTests/VersionSpecificTests.cs

```diff
@@ -29,9 +30,14 @@ public static IEnumerable<object[]> GetR4OnlyResources()
             yield return new object[] { "R4OnlyResource/ServiceRequest.json", "R4OnlyResource/ServiceRequest-target.json" };
         }
 
-        public static IEnumerable<object[]> GetCommonResourcesWithStu3OnlyField()
+        public static IEnumerable<object[]> GetCommonResourcesWithStu3OnlyValue()
         {
             yield return new object[] { "Stu3OnlyResource/Claim-Stu3.json" };
+
+        }
+
+        public static IEnumerable<object[]> GetCommonResourcesWithStu3OnlyElement()
```

Comment:
Just curious what's the difference between `value` and `element`?

Reply:
> Just curious what's the difference between `value` and `element`?

'element' is the type e.g. 'Name', 'value' is the detail value e.g. 'Jason', the updated version returns different exception types for unsupported 'element' and 'value' format while the old version just returns the same exception, so we need to split the unit test now.

Reply:
Got it. Thanks!